code/__DEFINES/genetics.dm

```diff
@@ -27,6 +27,7 @@
 
 // Extra powers:
 #define LASER			9 	// harm intent - click anywhere to shoot lasers from eyes
+#define GREY			10	//Wingdings for Greys -- Not activatable or researchable
```

Comment:
The problem is checking a player's species each time he talks is iffier than checking for a mutation. Note that this mutation is the same as LASER eyes e.g. not togglable via Player Panel or researchable through genetics.